docs/Server.md

```diff
@@ -616,8 +616,16 @@ Fake http injection (for testing purposes) [here](https://github.com/fastify/fas
 
 <a name="add-schema"></a>
 #### addSchema
-`fastify.addSchema(schemaObj)`, adds a shared schema to the Fastify instance. This allows you to reuse it everywhere in your application just by writing the schema id that you need.<br/>
-To learn more, see [shared schema example](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#shared-schema) in the [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md) documentation.
+`fastify.addSchema(schemaObj)`, adds a JSON schema to the Fastify instance. This allows you to reuse it everywhere in your application just by using the standard `$ref` keyword.<br/>
+To learn more, checkout the [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md) documentation.
+
+<a name="get-schemas"></a>
+#### getSchemas
+`fastify.getSchemas()`, returns a JSON with all the schemas added with `.addSchema`. The keys of the JSON are the `$id`s of the JSON schema provided.
```

Comment:
What is "a JSON"? Are you saying this method returns one big object with all schemas attached to properties?

Reply:
Here the case:

```
fastify.addSchema({ $id: 'A', properties: { hello: '...' } })
fastify.addSchema({ $id: 'B', properties: { hello: '...' } })

fastify.getSchemas() // will returns
  {
    A: { $id: 'A', properties: { hello: '...' } },
    B: { $id: 'B', properties: { hello: '...' } }
  }
```

Could you be clearer than me to write this behaviour? 😅

Reply:
```suggestion
`fastify.getSchemas()`, returns a hash of all schemas added via `.addSchema`. The keys of the hash are the `$id`s of the JSON Schema provided.
```